fe/fe-core/src/main/java/org/apache/doris/analysis/InlineViewRef.java

```diff
@@ -186,7 +189,13 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
         }
 
         if (view == null && !hasExplicitAlias()) {
```

Comment:
this is only for old optimizer, if support new optimizer, should also add logical in `org.apache.doris.nereids.parser.LogicalPlanBuilder#visitAliasedQuery`

Reply:
if meet inline view without alias, return '_', is this consistent to hive ?